transport/http/src/main/java/org/kaazing/gateway/transport/http/HttpUtils.java

```diff
@@ -283,7 +285,8 @@ private static boolean hasBeenModified(File requestFile,
 		}
 
 		// check modified time against file last modified
-		return lastModified > ifModifiedSinceDate.getTime();
+		double time_difference = Math.floor(Math.abs(lastModified - ifModifiedSinceDate.getTime())/1000);
```

Comment:
Consider using static imports, I personally don't and sometimes find static import annoying, but our code base seems to be trending in that direction